Packs/Mimecast/Integrations/MimecastV2/MimecastV2.py

```diff
@@ -556,7 +1017,7 @@ def set_empty_value_args_policy_update(policy_obj, option, policy_id):
      """
     empty_args_list = []
     # Add the empty arguments to empty args list
-    for arg, value in policy_obj.items():
+    for arg, value in list(policy_obj.items()):
```

Comment:
```suggestion
    for arg, value in policy_obj.items():
```
I guess 2to3 added this, I think it's redundant